.github/actions/ginkgo/main-k8s-versions.yaml

```diff
@@ -6,25 +6,29 @@ include:
     # renovate: datasource=docker
     kube-image: "quay.io/cilium/kindest-node:v1.32.0@sha256:22cf2864f90cfab0d442fda2decf2eae107edd03483053a902614dec637eff76"
     # renovate: datasource=docker depName=quay.io/lvh-images/kind
-    kernel: "bpf-next-20250319.212144"
+    kernel: "6.12-20250226.175419"
+    kernel-type: "latest"
 
   - k8s-version: "1.31"
     ip-family: "dual"
     # renovate: datasource=docker
     kube-image: "quay.io/cilium/kindest-node:v1.31.0@sha256:d2b2a8cd6fa282b9a4126938341a4d2924dfa96f60b1f983d519498c9cde1a99"
     # renovate: datasource=docker depName=quay.io/lvh-images/kind
     kernel: "rhel8.6-20250319.212144"
+    kernel-type: "oldstable"
```

Comment:
It would be nice to leave a comment of what does "oldstable" vs "latest" vs "stable" means.

Reply:
I would if I knew how we initially picked those kernels :sweat_smile: